decidim-core/app/assets/stylesheets/decidim/modules/_cards.scss

```diff
@@ -413,6 +413,10 @@ $datetime-bg: $primary;
   display: none;
   fill: $anchor-color;
 
+  &.alert{
```

Comment:
There's a [mixin](https://github.com/decidim/decidim/blob/f189d9305ecddcbc45a4d2ddaff6f4fbb5a4c837/decidim-core/app/assets/stylesheets/decidim/utils/_mixins.scss#L46) called `modifiers`, that you might use to wrap all possibilities (even they're not in use).
Instead of this block, switch it by `@include modifiers(fill);`. Now, no matter which status you assign to the parent element, the property `fill` will change accordingly

Reply:
Extra tip: I miss `$anchor-color` is a custom color status, so in that case.... `@include modifiers(fill, (alert: $anchor-color))`. Now I see again, it's not a big deal, but it's a way to handle statuses in the same way. You'll find other examples in the same file